test/integration_test/applicationbackup_test.go

```diff
@@ -251,96 +271,129 @@ func getBackupFromListWithAnnotations(backupList *storkv1.ApplicationBackupList,
 }
 
 func applicationBackupRestoreTest(t *testing.T) {
+	// Get location types and secret from config maps
+	configMap, err := k8s.Instance().GetConfigMap(configMapName, "default")
```

Comment:
`configMap` isn't being used in any of the tests, right?

Reply:
deleted